diff --git a/node_modules/qrcode/lib/utils/typedarray-buffer.js b/node_modules/qrcode/lib/utils/typedarray-buffer.js
index 9cde441..b39667c 100644
--- a/node_modules/qrcode/lib/utils/typedarray-buffer.js
+++ b/node_modules/qrcode/lib/utils/typedarray-buffer.js
@@ -336,7 +336,8 @@ Buffer.prototype.write = function write (string, offset, length) {
   return utf8Write(this, string, offset, length)
 }
 
-Buffer.prototype.slice = function slice (start, end) {
+if (!Buffer.prototype.slice) {
+Object.defineProperty(Buffer.prototype, 'slice', function slice (start, end) {
   var len = this.length
   start = ~~start
   end = end === undefined ? len : ~~end
@@ -371,6 +372,7 @@ Buffer.prototype.slice = function slice (start, end) {
   }
 
   return newBuf
+});
 }
 
 Buffer.prototype.copy = function copy (target, targetStart, start, end) {
