diff --git a/node_modules/react-native-document-picker/index.js b/node_modules/react-native-document-picker/index.js
index c87eb62..95ab12c 100644
--- a/node_modules/react-native-document-picker/index.js
+++ b/node_modules/react-native-document-picker/index.js
@@ -2,7 +2,7 @@
 import { Platform, NativeModules } from 'react-native';
 const { RNDocumentPicker } = NativeModules;
 
-if (!RNDocumentPicker) {
+if (!RNDocumentPicker && process.env.NODE_ENV !== 'test') {
   // Use a timeout to ensure the warning is displayed in the YellowBox
   setTimeout(() => {
     console.warn(
diff --git a/node_modules/react-native-document-picker/ios/RNDocumentPicker/RNDocumentPicker.m b/node_modules/react-native-document-picker/ios/RNDocumentPicker/RNDocumentPicker.m
index 1cb8b4b..eead541 100644
--- a/node_modules/react-native-document-picker/ios/RNDocumentPicker/RNDocumentPicker.m
+++ b/node_modules/react-native-document-picker/ios/RNDocumentPicker/RNDocumentPicker.m
@@ -5,6 +5,7 @@
 #if __has_include(<React/RCTConvert.h>)
 #import <React/RCTConvert.h>
 #import <React/RCTBridge.h>
+#import <React/RCTUtils.h>
 #else // back compatibility for RN version < 0.40
 #import "RCTConvert.h"
 #import "RCTBridge.h"
@@ -72,10 +73,7 @@ - (dispatch_queue_t)methodQueue
     }
 #endif
     
-    UIViewController *rootViewController = [[[[UIApplication sharedApplication]delegate] window] rootViewController];
-    while (rootViewController.presentedViewController) {
-        rootViewController = rootViewController.presentedViewController;
-    }
+    UIViewController *rootViewController = RCTPresentedViewController();
     
     [rootViewController presentViewController:documentPicker animated:YES completion:nil];
 }
