diff --git a/node_modules/react-native/Libraries/Modal/Modal.js b/node_modules/react-native/Libraries/Modal/Modal.js
index 08a5d74..65c41e0 100644
--- a/node_modules/react-native/Libraries/Modal/Modal.js
+++ b/node_modules/react-native/Libraries/Modal/Modal.js
@@ -19,8 +19,9 @@ const React = require('React');
 const PropTypes = require('prop-types');
 const StyleSheet = require('StyleSheet');
 const View = require('View');
+const requireNativeComponent = require('requireNativeComponent');
 
-const RCTModalHostView = require('RCTModalHostViewNativeComponent');
+const RCTModalHostView = Platform.OS === 'ios' ? requireNativeComponent('RCTModalHostView') : requireNativeComponent('TranslucentModalHostView');
 
 const ModalEventEmitter =
   Platform.OS === 'ios' && NativeModules.ModalManager
