diff --git a/node_modules/realm/src/jsc/jsc_value.hpp b/node_modules/realm/src/jsc/jsc_value.hpp
index 5c113e5..6b74ec2 100644
--- a/node_modules/realm/src/jsc/jsc_value.hpp
+++ b/node_modules/realm/src/jsc/jsc_value.hpp
@@ -50,6 +50,9 @@ inline const char *jsc::Value::typeof(JSContextRef ctx, const JSValueRef &value)
         case kJSTypeString: return "string";
         case kJSTypeBoolean: return "boolean";
         case kJSTypeUndefined: return "undefined";
+        #if defined __IPHONE_12_2 || defined __MAC_10_14_4
+            case kJSTypeSymbol: return "symbol";
+        #endif
     }
 }
 
diff --git a/node_modules/realm/src/rpc.cpp b/node_modules/realm/src/rpc.cpp
index 0c7bce4..e830b63 100644
--- a/node_modules/realm/src/rpc.cpp
+++ b/node_modules/realm/src/rpc.cpp
@@ -552,6 +552,9 @@ json RPCServer::serialize_json_value(JSValueRef js_value) {
         case kJSTypeString:
             return {{"value", jsc::Value::to_string(m_context, js_value)}};
         case kJSTypeObject:
+        #if defined __IPHONE_12_2 || defined __MAC_10_14_4
+            case kJSTypeSymbol:
+        #endif
             break;
     }
 
